04/app.js

```diff
@@ -2,8 +2,27 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
-        year: '1985'
+        day: '21',
+        month: '03',
+        year: '1984'
     }
-}
\ No newline at end of file
+}
+
+user.getBirthdayDay = function() {
+
+    const today = new Date().toLocaleDateString();
```

Comment:
To rozwiązanie jest o tyle problematycznie, że w zależności od ustawień systemowych dana może być inaczej reprezentowana: https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString